probe/endpoint/conntrack_test.go

```diff
@@ -0,0 +1,138 @@
+package endpoint_test
+
+import (
+	"bufio"
+	"encoding/xml"
+	"io"
+	"testing"
+	"time"
+
+	. "github.com/weaveworks/scope/probe/endpoint"
+	"github.com/weaveworks/scope/test"
+	"github.com/weaveworks/scope/test/exec"
+)
+
+func makeFlow(id int64, srcIP, dstIP string, srcPort, dstPort int, ty, state string) Flow {
+	return Flow{
+		XMLName: xml.Name{
+			Local: "flow",
+		},
+		Type: ty,
+		Metas: []Meta{
+			{
+				XMLName: xml.Name{
+					Local: "meta",
+				},
+				Direction: "original",
+				Layer3: Layer3{
+					XMLName: xml.Name{
+						Local: "layer3",
+					},
+					SrcIP: srcIP,
+					DstIP: dstIP,
+				},
+				Layer4: Layer4{
+					XMLName: xml.Name{
+						Local: "layer4",
+					},
+					SrcPort: srcPort,
+					DstPort: dstPort,
+					Proto:   TCP,
+				},
+			},
+			{
+				XMLName: xml.Name{
+					Local: "meta",
+				},
+				Direction: "independent",
+				ID:        id,
+				State:     state,
+				Layer3: Layer3{
+					XMLName: xml.Name{
+						Local: "layer3",
+					},
+				},
+				Layer4: Layer4{
+					XMLName: xml.Name{
+						Local: "layer4",
+					},
+				},
+			},
+		},
+	}
+}
+
+func TestConntracker(t *testing.T) {
+	oldExecCmd := exec.Command
+	defer func() { exec.Command = oldExecCmd }()
+
+	reader, writer := io.Pipe()
+	exec.Command = func(name string, args ...string) exec.Cmd {
+		return exec.NewMockCmd(reader)
+	}
+
+	conntracker, err := NewConntracker()
+	if err != nil {
+		t.Fatal(err)
```

Comment:
```
ugh ~/src/github.com/weaveworks/scope/probe/endpoint (356-conntrack) go test -v -test.run=TestConntracker
=== RUN   TestConntracker
--- FAIL: TestConntracker (0.00s)
        conntrack_test.go:76: No conntrack module
FAIL
exit status 1
```

Please put this test behind an OS flag somehow. Maybe call the file conntracker_linux_test.go?